testcases/GcovSetup.py

```diff
@@ -150,6 +154,8 @@ def add_gcov_param(self, conf_file):
                 log.info("param failed to change {param}")
                 err_param.append(param)
             log.info("\n\n\n")
+        if self.distro_name == 'sles':
+            self.cv_HOST.host_run_command(f"sed -i 's/^.*CONFIG_SYSTEM_TRUSTED_KEYS/#&/g' {conf_file}")
```

Comment:
the above config option is needed for both SLES and RHEL. can you make it generic

Reply:
no, it only for SLES

Reply:
so the rhel is rpmbuild way